<!-- /-->
<launch>
  <node pkg="lawn_mower_pattern" type="lawn_mower_pattern_generator" name="lawn_mower_pattern">
		  <param name="work_frame" value="/world" />
		  <param name="trajectory_topic" value="/local_planner/trajectory" /> 
		  <param name="visualization_topic" value="coverage_trajectory" />
		  <param name="odo_topic" value="/mk_model/odometry" /> 
		  <param name="publish_trajectory_topic" value="publish_trajectory" />

		  <param name="r" value="0.0"/>
		  <param name="g" value="1.0" /> 
		  <param name="b" value="0.0" />
		  <param name="a" value="1.0" />
		  <param name="scale" value="0.1" />
		  <param name="namespace" value="trajectory" />

		  <param name="box_x" value="50.0"/>
		  <param name="box_y" value="15.0" /> 
		  <param name="row_distance" value="5.0" />
		  <param name="temporal_res" value="0.1" />
		  <param name="radius" value="2.0" />
		  <param name="velocity" value="10.0" />
		  <param name="acceleration" value="2.0" />
		  <param name="altitude" value="0.0" /> 
		  <param name="constant_heading" value="false" /> 
   </node>
</launch>
